src/state_management.cpp

```diff
@@ -57,6 +57,10 @@ namespace ara::sm {
         if (stateClient != nullptr) {
             internalState = stateClient->SmGetState();
         }
+        if (executionClient != nullptr) {
+            executionClient->ReportApplicationState(internalState);
+        }
+        triggerOut.SetNotifier(internalState);
```

Comment:
triggered, this is already called in Work - delete it from there, or from here.

Reply:
Done